k8s_itests/Makefile

```diff
@@ -0,0 +1,16 @@
+# You need kind installed to run this!
+
+CLUSTER  := paasta-k8s-test-cluster
```

Comment:
kind clusters are global, use `$USER` to isolate

Reply:
Nice catch. I set the cluster to a fixed name because it's painful to dynamically change the cluster names in the assets.

Reply:
the issue is that if two people on same machine run these itests their clusters will conflict, this is why i think we should use $USER